axelrod/strategies/grudger.py

```diff
@@ -342,3 +342,75 @@ def strategy(opponent: Player) -> Action:
         elif opponent.defections:
             return D
         return C
+
+
+class Capri(Player):
+    """
+    CAPRI is a memory-3 strategy proposed in [Murase2020]_. Its behavior is defined by the following five rules.
+    C: Cooperate at mutual cooperation.
+        This rule prescribes c at (ccc, ccc).
+    A: Accept punishment when you mistakenly defected from mutual cooperation.
+        This rule prescribes c at (ccd, ccc), (cdc, ccd), (dcc, cdc), and (ccc, dcc).
+    P: Punish your co-player by defecting once when he defected from mutual cooperation.
+        This rule prescribes d at (ccc, ccd), and then c at (ccd, cdc), (cdc, dcc), and (dcc, ccc).
+    R: Recover cooperation when you or your co-player cooperated at mutual defection.
+        This rule prescribes c at (ddd, ddc), (ddc, dcc), (dcc, ccc), (ddc, ddd), (dcc, ddc), (ccc, dcc), (ddc, ddc), and (dcc, dcc).
+    I: In all the ohter cases, defect.
```

Comment:
> Thank you for pointing that out. Could you tell me how to test the docstring?

Of course: top build the documentation, `cd docs` (go to the `docs` directory) and run:

```
make html
```

You might need to `pip` install some dependencies.

I have checked locally and below fixes the sphinx build problems:

```suggestion
    CAPRI is a memory-3 strategy proposed in [Murase2020]_. Its behavior is
    defined by the following five rules.

    - C: Cooperate at mutual cooperation.  This rule prescribes c at (ccc, ccc).
    - A: Accept punishment when you mistakenly defected from mutual cooperation.
      This rule prescribes c at (ccd, ccc), (cdc, ccd), (dcc, cdc), and (ccc,
      dcc).
    - P: Punish your co-player by defecting once when he defected from mutual
      cooperation.  This rule prescribes d at (ccc, ccd), and then c at (ccd,
      cdc), (cdc, dcc), and (dcc, ccc).
    - R: Recover cooperation when you or your co-player cooperated at mutual
      defection.  This rule prescribes c at (ddd, ddc), (ddc, dcc), (dcc, ccc),
      (ddc, ddd), (dcc, ddc), (ccc, dcc), (ddc, ddc), and (dcc, dcc).
    - I: In all the ohter cases, defect.

    Names:

    - CAPRI: Original Name by Y. Murase et al. [Murase2020]_

```

Reply:
nit: ohter -> other on line 362